presto-main/src/main/java/com/facebook/presto/execution/NodeResourceStatusConfig.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.execution;
+
+import com.facebook.airlift.configuration.Config;
+import com.facebook.airlift.configuration.ConfigDescription;
+
+import javax.validation.constraints.Min;
+
+public class NodeResourceStatusConfig
+{
+    private int requiredWorkersActive;
+
+    @Min(0)
+    public int getRequiredWorkersActive()
+    {
+        return requiredWorkersActive;
+    }
+
+    @Config("query-manager.required-workers-active")
```

Comment:
we should call this something like `cluster.required-workers-active` as it's for the cluster and not really for query manager.